creator-node/src/routes/nodeSync.js

```diff
@@ -95,40 +141,52 @@ module.exports = function (app) {
       params: { wallet_public_key: walletPublicKeys },
       responseType: 'json'
     })
-    if (resp.status !== 200) return errorResponse(resp.status, resp.data['error'])
-    if (!resp.data.hasOwnProperty('cnodeUsers')) {
-      return errorResponseBadRequest(`Malformed response from ${creatorNodeEndpoint}. "cnodeUsers" property not found on response object.`)
+    if (resp.status !== 200) throw new Error(resp.data['error'])
+    // TODO - explain patch
+    if (!resp.data) {
+      if (resp.request && resp.request.responseText) {
+        resp.data = JSON.parse(resp.request.responseText)
+      } else throw new Error(`Malformed response from ${creatorNodeEndpoint}.`)
+    }
+    if (!resp.data.hasOwnProperty('cnodeUsers') || !resp.data.hasOwnProperty('ipfsIDObj') || !resp.data.ipfsIDObj.hasOwnProperty('addresses')) {
+      throw new Error(`Malformed response from ${creatorNodeEndpoint}.`)
     }
 
     // Attempt to connect directly to target CNode's IPFS node.
-    if (resp.data.hasOwnProperty('ipfsIDObj') && resp.data.ipfsIDObj.hasOwnProperty('addresses')) {
-      await _initBootstrapAndRefreshPeers(req, resp.data.ipfsIDObj.addresses, redisKey)
-    } else {
-      return errorResponseBadRequest(`Malformed response from ${creatorNodeEndpoint}. Malformatted or missing "ipfsIDObj" property.`)
-    }
+    await _initBootstrapAndRefreshPeers(req, resp.data.ipfsIDObj.addresses, redisKey)
```

Comment:
added this just for logging while devving nodesync. useful to see which wallet was being synced since multiple were running concurrently. pretty janky for sure